test/e2e/common/pods.go

```diff
@@ -44,6 +44,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2ekubelet "k8s.io/kubernetes/test/e2e/framework/kubelet"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	websocketutil "k8s.io/kubernetes/test/e2e/framework/websocket"
```

Comment:
Follow the naming pattern like `e2ewebsocket`

Reply:
+1
It is nice to add it into `./hack/.import-aliases` also.

Reply:
sure i will update it

Reply:
There are 2 alias name pattern in e2e. `e2efoo` or `e2efooutil`. Which is better?